internal/logging/gorm.go

```diff
@@ -47,6 +47,6 @@ func (l *gormLogger) Error(ctx context.Context, s string, args ...interface{}) {
 }
 
 func (l *gormLogger) Trace(ctx context.Context, begin time.Time, fc func() (sql string, rowsAffected int64), err error) {
-	// don't log traces for now
-	_, _ = fc()
+	sql, rows := fc()
+	l.l.Debugf("SQL (%d rows, %s): %s", rows, time.Since(begin).String(), sql)
```

Comment:
minor optimization: we can avoid doing some work when debug logging is off by removing the `String()`. The logger will call that for us when debug logs are enabled, but the work is skipped when debug logs are off.

```suggestion
	l.l.Debugf("SQL (%d rows, %s): %s", rows, time.Since(begin), sql)
```

We could go even further and optimize away the `time.Since()` call in that case with our own type, but doesn't seem worth it right now.